jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/logging/AnsiLoggerWithFooter.java

```diff
@@ -103,13 +116,16 @@ public void log(Level logLevel, String message) {
 
     singleThreadedExecutor.execute(
         () -> {
-          if (eraseFooter()) {
-            // If a previous footer was erased, the message needs to go up a line. However, we go up
-            // two lines (and then down one line by calling "accept()" once) to correct an issue in
-            // Maven: https://github.com/GoogleContainerTools/jib/issues/1952
+          boolean didErase = eraseFooter();
+
+          // If a previous footer was erased, the message needs to go up a line. But only do so when
+          // not doing the two-cursor-up fix.
+          String prefix = didErase && !enableTwoCursorUpJump ? CURSOR_UP_SEQUENCE : "";
```

Comment:
Maybe personal preference, but I think the shorter code here is harder to follow than just separating each condition fully.
```
if (didErase) {
  if (enableTwoCursorUpJump) {
    messageConsumer.accept(String.format(CURSOR_UP_SEQUENCE_TEMPLATE, 2));
    messageConsumer.accept(message);
  } else {
    messageConsumer.accept(CURSOR_UP_SEQUENCE + message);
  }
} else {
  messageConsumer.accept(message);
}
```

Unfortunately that leads to even more duplication.

Reply:
Yeah, that's true. Expanded the code.